### PR TITLE
feat: add fedimint `OOBNotes`

### PR DESCRIPTION
**Pull request by oleonardolima**

### CODE REVIEW

**Comment by oleonardolima** [comment]
~~note: my devimint/mprocs is broken for some reason, fixing that ill fix the test and it's good to go.~~

**Comment by oleonardolima** [comment]
when upgrading to v.0.3 the same OOB notes could be used to get the invitation code as well

https://github.com/fedimint/fedimint/blob/master/modules/fedimint-mint-client/src/lib.rs#L97-L104

File: waila/src/lib.rs [hunk 1]
```diff
@@ -847,6 +886,25 @@ mod tests {
...
+    fn parse_fedimint_oob_notes() {
+        let parsed = PaymentParams::from_str(SAMPLE_FEDIMINT_OOB_NOTES).unwrap();
+
+        assert_eq!(parsed.address(), None);
+        assert_eq!(parsed.memo(), None);
+        assert_eq!(parsed.network(), None);
+        assert_eq!(parsed.invoice(), None);
+        assert_eq!(parsed.node_pubkey(), None);
+        assert_eq!(parsed.amount(), Some(Amount::from_sat(10)));
+        // NOTE: (@leonardo) there is not `Eq` implementation for `fedimint-mint-client::OOBNotes`
```
Comment (benthecarman) [inline waila/src/lib.rs:899]: added: https://github.com/fedimint/fedimint/pull/4346

Comment (oleonardolima) [reply waila/src/lib.rs:899]: nice! thx!

Comment (oleonardolima) [reply waila/src/lib.rs:899]: @benthecarman would be a problem using a different release/commit here in waila when comparing it to mutiny-node ? 🤔

Comment (benthecarman) [reply waila/src/lib.rs:899]: We should keep them in sync

Comment (oleonardolima) [reply waila/src/lib.rs:899]: ok, so we'll need to fix this when bumping the mutiny-node to a new release

Comment (benthecarman) [reply waila/src/lib.rs:899]: Yeah that's fine. Hopefully one day it'll all just be on 0.3

**Review by benthecarman — APPROVED** [review]

**Pull request merged**